gammapy/modeling/tests/test_sampling.py

```diff
@@ -52,3 +54,20 @@ def test_runmcmc(dataset):
 
     sampler = run_mcmc(dataset, nwalkers=6, nrun=10)  # to speedup the test
     assert isinstance(sampler, emcee.ensemble.EnsembleSampler)
+
```

Comment:
Please remove the empty lines here. The PEP8 standard recommends just 2 empty lines between two function definitions...